evm/src/arithmetic/mod.rs

```diff
@@ -122,6 +126,9 @@ pub(crate) enum Operation {
         input2: U256,
         result: U256,
     },
+    RangeCheckOperation {
+        values: [u32; NUM_CPU_LIMBS],
+    },
```

Comment:
Please make it `U256` for consistency. Every `U256` that the arithmetic table deals with is composed of eight `u32` limbs, so it's the same.